### PR TITLE
NR - Update Add New Family form

### PR DESCRIPTION
**Pull request by tvcam**

### CODE REVIEW

**Comment by kirykr** [comment]
So if you use Vcode, you should use the plugin [ruby-rubocop](https://marketplace.visualstudio.com/items?itemName=misogi.ruby-rubocop) and if you use sublime you should use [SublimeLinterRubocop](https://packagecontrol.io/packages/SublimeLinter-rubocop) and this bellow is the sublime config
// SublimeLinter Settings - User
{
  "debug": true,
  "linters": {
    "rubocop": {
      "executable": "/Users/kiry/.rbenv/shims/rubocop",
      "excludes": ["**/*.js.erb"],
      "args": ["--config", ".rubocop.yml"]
    }
  },
  "paths": {
    "osx": [
      "~/.rbenv/shims"
    ]
  }
}
```
Noted: /Users/kiry/.rbenv/shims/rubocop  you might use RVM so you can find rubocop  by running which rubocop and use that path and see the path if you use linux change key to Linux and and use rvm path instead

**Comment by kirykr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/models/family.rb [hunk 1]
```diff
@@ -30,38 +47,15 @@ class Family < ActiveRecord::Base
   has_paper_trail
 
   before_validation :assign_family_type, if: [:new_record?, :brc?]
+  before_validation :assign_status
 
   validates :family_type, presence: true, inclusion: { in: TYPES }
-  validates :code, uniqueness: { case_sensitive: false }, if: 'code.present?'
-  validates :status, presence: true, inclusion: { in: STATUSES }
-  validate :client_must_only_belong_to_a_family
```
Comment (PirunSeng) [inline app/models/family.rb:37]: @tvcam is this validation not needed anymore bro?

Comment (tvcam) [reply app/models/family.rb:37]: Some validations are removed to satisfied the current feature, they will be reviewed again after before finalizing the PR @PirunSeng

Comment (tvcam) [reply app/models/family.rb:37]: @PirunSeng I've fixed/updated related specs, but travi-ci is out of quota, and cannot do more build

Comment (PirunSeng) [reply app/models/family.rb:37]: > 
> 
> Some validations are removed to satisfied the current feature, they will be reviewed again after before finalizing the PR @PirunSeng

Noted bro, I'm just afraid you removed it accidentally :). So, maybe you want to comment out or remove the method too.

**Pull request merged**